TourMate/TourMate/Frontend/Plan/Views/PlansViews/PlansMapDayView.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  PlansMapDayView.swift
+//  TourMate
+//
+//  Created by Rayner Lim on 11/4/22.
+//
+
+import SwiftUI
+import MapKit
+
+struct IdentifiableLocation: Identifiable {
+    var id: Int
+    var coordinate: CLLocationCoordinate2D
+    var status: PlanStatus
+}
+
+struct PlansMapDayView: View {
```

Comment:
Do you think it will be better if we have a `PlansMapDayViewModel` and manage all the states and logic from there?

Reply:
I guess so but I don't have time to work on this any more, I still have the drag and drop to try to do

Reply:
I'm not sure but I think it might not be as easy as shifting everything into a view model, things could break. The code looks quite simple but it took a long time to get everything working correctly so not sure how long that would take.